packages/web/pages/index.tsx

```diff
@@ -18,8 +18,19 @@ interface HomeProps {
   ads: Ad[];
 }
 
+const ADS_URL =
+  "https://raw.githubusercontent.com/osmosis-labs/fe-content/main/cms/swap-rotating-banner.json";
+
 export const getStaticProps: GetStaticProps<HomeProps> = async () => {
```

Comment:
I think we will need to use the `revalidate` option in `getStaticProps` here to bust the cache every once in a while. Maybe once an hour or something like that. Otherwise, the cache only updates on every rebuild (i.e. every deploy)

Reply:
Sounds good to me. We could always update it if need be.